src/app/modules/shared_models/message_model.nim

```diff
@@ -120,6 +124,10 @@ QtObject:
     case enumRole:
     of ModelRole.Id:
       result = newQVariant(item.id)
+    of ModelRole.PrevMsgIndex:
+      result = newQVariant(index.row + 1)
+    of ModelRole.NextMsgIndex:
+      result = newQVariant(index.row - 1)
```

Comment:
Shouldn't "next" be a `+1` and "prev" a `-1`?

Reply:
This is a good question :) @caybro there is a comment in ChatMessagesView about it.

Reply:
Ah, I see, the messages are in reverse order (newest at the bottom)